packages/components/src/icon/stories/index.story.tsx

```diff
@@ -99,3 +123,28 @@ WithADashicon.args = {
 	...Default.args,
 	icon: 'wordpress',
 };
+
+export const WithAnIconFromTheLibrary: StoryFn< IconProps > = ( args ) => {
```

Comment:
What was the motivation for adding this? I think we can improve it based on that. For example, I can think of two motivators:

- I want to show that we can use icons from the library. In this case, it would be more useful to have a description for the story, and an actual code snippet showing the imports.
- I want to show what icons there are in the library. In this case, it would be more useful to send people to the [icon library story](https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library), which shows the entire set with a search field.

Reply:
The motivation is to be able to render any icon of the library and play with the different props passed to that icon. That's neither possible in the Icon or the icon library story.